guides/plugins/apps/app-base-guide.md

```diff
@@ -131,6 +131,14 @@ $signature = hash_hmac('sha256', $queryString, $appSecret);
 
 ### Registration Response
 
+There may be valid cases where the app installation fails, because the domain is blocked, or some other prerequisite in that shop is not met, in which case you can return the message error as follows
+```javascript
+{
+  "error": "Shop url is not met"
```

Comment:
Meeting a URL is a phrase I don't know 🤔
Do you mean reachable or it didn't meet validation criteria? In the latter case I would call it "the shop URL is invalid"